### PR TITLE
feat: Engagement center Open program detail by clicking on program card - MEED-417 - Meeds-io/MIPs#13

### PR DESCRIPTION
**Pull request by AzmiTouil**
Open program detail by clicking on program card

### CODE REVIEW

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        min-width="70px"
+        width="100%"
+        max-height="140px"
+        class="primary--text border-color" />
+    </div>
+    <div class="pt-5">
+      <v-list-item two-line class="px-0">
+        <v-list-item-content>
+          <v-list-item-title class="text-color font-weight-bold">
+            Description
```
Comment (plamarque) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue]: i18n

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            </div>
+          </v-list-item-subtitle>
+        </v-list-item-content>
+      </v-list-item>
+    </div>
+    <div class="pt-5">
+      <v-list-item two-line class="px-0">
+        <v-list-item-content>
+          <v-list-item-title class="text-color font-weight-bold">
+            Rules of program
```
Comment (plamarque) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue]: i18n

**Review by plamarque — COMMENTED** [review]
don't forget i18n ;)

**Comment by AzmiTouil** [comment]
> don't forget i18n ;)

yep :)

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramCard.vue [hunk 3]
```diff
@@ -15,71 +15,70 @@ along with this program; if not, write to the Free Software Foundation,
...
+          </v-list>
+        </v-menu>
+      </v-row>
+    </div>
+    <div class="d-flex flex-grow-1 pa-0">
+      <v-img
+        :src="programCover"
+        :alt="$t('programs.cover.default')"
+        aspect-ratio="1"
+        min-height="70px"
```
Comment (SaraBoutej) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramCard.vue]: You can put values without 'px' for min-hight, min-width, max-height props.

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+          <v-list-item-title class="text-color font-weight-bold">
+            {{ $t('programs.details.label.rulesOfProgram') }}
+          </v-list-item-title>
+          <v-list-item-subtitle class="text-color pt-4">
+            <v-data-table
+              :headers="rulesHeaders"
+              :items="programRules"
+              :options.sync="options"
+              :server-items-length="totalSize"
+              hide-default-footer
```
Comment (SaraBoutej) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue]: can you please put props without new values at the end.

**Review by SaraBoutej — APPROVED** [review]

**Pull request merged**